examples/data/e2e_tests/test_jsons/stdNtDnTest.json

```
@@ -30,7 +30,7 @@
         "pluginName" : "DatasetComparison",
         "name": "DatasetComparison",
         "order" : 1,
-        "args" : ["--format", "parquet", "--new-path", "#{new-std-data-path}#", "--ref-path", "#{ref-std-data-path}#", "--keys", "name" ],
+        "args" : ["--format", "parquet", "--new-path", "#{new-std-data-path}#", "--ref-path", "#{ref-std-data-path}#", "--keys", "property", "--datetimeRebaseMode", "LEGACY"],
```

```suggestion
        "args" : ["--format", "parquet", "--new-path", "#{new-std-data-path}#", "--ref-path", "#{ref-std-data-path}#", "--keys", "property"],
```

I think we said this is not needed anymore

---

So we updated Hermes to include `datetimeRebaseMode`? 

We removed the opting afaik - https://github.com/AbsaOSS/hermes/pull/132

---

You are right, I was confused. `datetimeRebaseMode` removed in commit 92d983d65f9275664edbe8c3f8124b0ccd0b9561.